src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig

```diff
@@ -0,0 +1,18 @@
+<form id="theme_search" method="post" action="{{ path('search_user_search') }}{# @todo probably an incorrect path #}">
+    <div>
+        <input id="block_search_q" type="search" name="q" size="20" maxlength="255" results="10" autosave="Search" class="theme_search_input" />
+        {% if vars.displaySearchBtn == 1 %}
+        <input class="theme_search_button" type="submit" value="{gt text="Search" domain='zikula'}" />
```

Comment:
this was [addressed here](https://github.com/zikula/Tools/commit/5a7a78d97ab8b78a57829afa5068211c3d237717)